src/view/screens/Settings.tsx

```diff
@@ -354,12 +354,12 @@ export const SettingsScreen = withAuthRequired(
               />
             </View>
             <Text type="lg" style={dangerText}>
-              Delete my account
+              Delete my account…
```

Comment:
To follow the convention of system menu items etc. where ellipsis means there will be further prompts before any action is actually taken.